eval.cpp

```diff
@@ -784,6 +793,11 @@ namespace Sass {
       Expression* ex = operator()(var);
       return evacuate_quotes(interpolation(ex));
 
+    } else if (Parent_Selector* var = dynamic_cast<Parent_Selector*>(s)) {
+
+      Expression* ex = operator()(var);
```

Comment:
Yeah i was following convention there in that method but happy to change the rest of the calls.  Shall I?